packages/valory/skills/task_execution/behaviours.py

```diff
@@ -86,8 +86,7 @@ def __init__(self, **kwargs: Any):
         self._all_tools: Dict[str, Tuple[str, str, Dict[str, Any]]] = {}
         self._inflight_tool_req: Optional[str] = None
         self._done_task: Optional[Dict[str, Any]] = None
-        self._last_polling: Optional[float] = None
-        self._last_polling_marketplace: Optional[float] = None
+        self._req_params = self.params.req_params
```

Comment:
why not just access it directly in the params? i dont think we need the assignment here